dashboard/src/pages/login/CoreSelectExisting.tsx

```diff
@@ -133,6 +135,33 @@ const CoreSelectExisting = () => {
                   );
                 }}
               />
+              {
+                tauri && <Button
+                type="button"
+                iconRight={faScrewdriverWrench}
+                label="Setup Core Locally"
+                onClick={() => {
+                  // extremely sus setup that should probably be fixed by directly calling the onSubmit logic
+                  // rather than duplication
+                  const core = coreList[0]
```

Comment:
probably hard code or use the const localhost core.
coreList[0] is a little hacky